backendless.d.ts

```diff
@@ -752,6 +752,29 @@ declare module Backendless {
         toJSON(): DataQueryI;
     }
 
+    /**
+     * @public
+     * @class Backendless.JSONUpdateBuilder
+     * @constructor
+     */
+    class JSONUpdateBuilder {
+        static SET() : Backendless.JSONUpdateBuilder;
+
+        static INSERT() : Backendless.JSONUpdateBuilder;
+
+        static REPLACE() : Backendless.JSONUpdateBuilder;
+
+        static REMOVE() : Backendless.JSONUpdateBuilder;
+
+        static ARRAY_APPEND() : Backendless.JSONUpdateBuilder;
+
+        static ARRAY_INSERT() : Backendless.JSONUpdateBuilder;
+
+        addArgument(arg: string, argValue?: string) : this;
```

Comment:
I believe, for consistency,  here should be `Backendless.JSONUpdateBuilder` instead of `this`

Reply:
+